plugins/woocommerce-admin/client/tasks/fills/experimental-import-products/importTypes.tsx

```diff
@@ -14,14 +17,24 @@ export const importTypes = [
 			'woocommerce'
 		),
 		before: <PageIcon />,
+		href: getAdminLink(
+			'edit.php?post_type=product&page=product_importer&wc_onboarding_active_task=products'
+		),
 	},
 	{
 		key: 'from-cart2cart' as const,
```

Comment:
For "FROM CART2CART", only the "Learn more" link is clickable. But for "FROM A CSV FILE", the entire card is clickable. 

Is this intended?

![Screen Recording 2022-05-03 at 11 32 09](https://user-images.githubusercontent.com/4344253/166401657-cc0c7d7c-d7e0-4ea1-a0ad-18803e86bd49.gif)

Reply:
cc @verofasulo

Reply:
> For "FROM CART2CART", only the "Learn more" link is clickable. But for "FROM A CSV FILE", the entire card is clickable.

No, both the cards are entirely clickable :) We added the link 'Learn more" after conducting some tests as it reassures the users they are not starting any flow but only collecting information about this option. @chihsuan

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/32835/commits/84fe952ba81cae7fd016963b8f422e1721053419